File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/plagiarized/L3/04/Kasus2.kt
package IR

import java.util.Scanner

/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
fun main() {
    val input = Scanner(System.`in`)

    // Enter radius of the cylinder
    print("Enter the radius and length of a cylinder: ")
    val radius = input.nextDouble()
    val length = input.nextDouble()

    val area = radius * radius * 3.14159
    val volume = area * length

    println("The area is $area")
    println("The volume of the cylinder is $volume")
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L1/04/Kasus2.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

console.log("Enter the radius and length of a cylinder: ");

rl.on('line', (input) => {
  const [radius, length] = input.split(' ').map(Number);
  const area = radius * radius * 3.14159;
  const volume = area * length;

  console.log(`The area is ${area}`);
  console.log(`The volume of the cylinder is ${volume}`);

  rl.close();
});

